src/runtimes/docker/clients/DockerClientBase/withDockerExposesArg.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { withNamedArg } from "../../utils/commandLineBuilder";
+
+export function withDockerExposesArg(ports?: Array<number>) {
+    return withNamedArg('--expose', ports.toString(), { shouldQuote: false });
```

Comment:
You'll want to keep the ports argument as an array, otherwise you'll get an error if multiple ports are passed in:

```suggestion
    return withNamedArg('--expose', (ports || []).map(port => port.toString()), { shouldQuote: false });
```

Reply:
that's right... rookie mistake :disappointed_relieved:

Reply:
It's an easy mistake to make with any kind of array data; happens to us all.